src/net_processing.cpp

```diff
@@ -3890,6 +3891,12 @@ bool PeerLogicValidation::ProcessMessages(CNode* pfrom, std::atomic<bool>& inter
     }
     CNetMessage& msg(msgs.front());
 
+    TRACE4(PeerLogicValidation, ProcessMessages,
```

Comment:
I think we'll want to add the peer id as well here (same for outgoing)

Reply:
done